cpp/meetings.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
+
+bool mycmp(pair<int, int> a, pair<int, int> b)
+{
+    /*
+    반례
+    3
+    4 4
+    3 4
+    2 4
+    */
+   if (a.second == b.second) return a.first < b.first;
+   return a.second < b.second;
+}
```

Comment:
3항 연산자를 활용할 수 있는 부분:
```suggestion
bool mycmp(pair<int, int> a, pair<int, int> b)
{
   return (a.second == b.second) ? (a.first < b.first) : (a.second < b.second);
}
```